scripts/node_control_tool.py

```diff
@@ -14,30 +19,41 @@ def compose_cmd(cmd):
 
 def call_upgrade_script(version):
     import subprocess
-    print('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
-    try:
-        if test_mode:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
-        else:
-            retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node', version]), shell=True)
-        if retcode != 0:
-            print('Upgrade failed')
-    except:
-        print('Something went wrong with calling upgrade script')
+    logger.info('Upgrading sovrin node to version %s, test_mode %d ' % (version, int(test_mode)))
+    if test_mode:
+        retcode = subprocess.call(compose_cmd(['upgrade_sovrin_node_test', version]), shell=True)
```

Comment:
Apt-get does not require root (uid==0). It requires elevated privileges (possibly euid==0). In the past 5 years, I have probably run apt-get 1000 times--5 as root, and 995 via sudo. I believe that sometimes I have run with sudo via a user who had limited privs, not ALL in the sudoers file, which would mean it is possible to create a user that just has the ability to run "sudo apt-get ..." in the sudoers file, nothing else.

You are correct that this is a different issue--but I am irritated that we are being sloppy about privileges. Please log a separate ticket.